### PR TITLE
Adding compatibility information between recommendations, Magento, and Page Builder

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) adds information about creating recommendations for Page Builder and which version of Magento supports recommendations with Page Builder.

If Magento Commerce, please specify:

- [x] Commerce only?
- [ ] Commerce with B2B?

## Affected documentation pages

- https://docs.magento.com/user-guide/marketing/create-new-rec.html
- https://docs.magento.com/user-guide/marketing/page-builder-add-product-recs.html

### CODE REVIEW

File: src/marketing/page-builder-add-product-recs.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ title: Add Product Recommendations
 
 Use the Product Recommendations content type to add an existing, active [recommendation unit]({% link marketing/create-new-rec.md %}) to a page, block, or dynamic block.
 
+{:.bs-callout-info}
+Product Recommendations are compatible with Page Builder starting in Magento 2.3.0, but not supported.
```
Comment (misha-kotov) [inline src/marketing/page-builder-add-product-recs.md:9]: this makes it sound like PB is not supported "starting in Magento 2.3.0". can we clarify that it's only that version, and the rest are supported?

Comment (leslietilling) [reply src/marketing/page-builder-add-product-recs.md:9]:
```suggestion
The Product Recommendations content type in Page Builder is supported in Magento 2.3.1 and later. In Magento 2.3.0, is compatible but not supported.
```

File: src/marketing/create-new-rec.md [hunk 2]
```diff
@@ -16,6 +16,9 @@ After you create and activate the recommendation unit, Magento will begin [colle
 
 1. Specify the [Store View]({% link configuration/scope-change.md %}) where you want the recommendations to display.
 
+   {:.bs-callout-info}
+   You only can create recommendation units for the Page Builder page type in the default store view.
```
Comment (leslietilling) [inline src/marketing/create-new-rec.md:20]:
```suggestion
   You can create recommendation units for the Page Builder page type only in the default store view.
```

**Comment by erikmarr** [comment]
running tests

**Pull request merged**